### ISSUE TITLE
time_window and until parameter used together?

### ISSUE THREAD

**Issue by dwrobel** [klen/graphite-beacon#141]
I have question related to using both **time_window** and **until** paremeters. The **time_window** is defined as following:

> // Default time window for Graphite queries
>         // Defaults to query interval, can be redefined for each alert.
>         "time_window": "10minute",

and is working as described as long as **until** parameter is *zero*.

But as soon as I start using **until** parameter with value a different than *zero* then requests send to the Graphite started to have incorrect **from** parameter not reflecting the expected time range defined by **time_window** parameter.

In other words once someone start to use **until** then **time_window** is reduced by the value of **until** parameter which is quite confusing as in that case **time_window** is no longer a time window.

Is my understanding correct that once **until** is in use with **time_window** then also **from** parameter passed to Graphite should be adjusted accordingly.
